doc/index.rst

```diff
@@ -1,8 +1,45 @@
-Plutus
-======
+Plutus Programming and Reference Guide
+=============================================
+
+About Plutus
+--------------------
+
+Plutus consists of Plutus Core, the programming, scripting and assembly language; 
+and Plutus Tx, the compiler that compiles the Haskell source code into Plutus Core 
+to form the on-chain part of a contract application. All of this is used in combination 
+to write Plutus scripts that run on the Cardano blockchain. Compiled Plutus Core 
+scripts or programs are able to interact with the Cardano ledger through the 
+ledger interface. 
+
+The Plutus Core Repository
+----------------------------------
+
+The `Plutus Core Repository <https://github.com/input-output-hk/plutus>`_ contains 
+the implementation, specification, and mechanized metatheory of Plutus Core. 
+It also contains Plutus Tx. 
+
+About This Documentation
+---------------------------------
+
+The purpose of this documentation is to serve as an introduction to the Plutus language 
+and to programming in Plutus. It is meant to be educational, so it includes explanations, 
+tutorials and how-to instructions. It is also intended as a reference for beginning and 
+experienced developers. 
+
+Intended Audience
+---------------------------
+
+The intended audience of this documentation includes DApp developers, Plutus developers, 
+developers writing smart contracts, Haskell developers who want to write Plutus scripts, 
+and developers who want to learn to use the compiler, Plutus Tx. It is also designed for 
+developers who want to learn Plutus directly from the team that developed the Plutus language. 
```

Comment:
Not sure what this means.

Reply:
these all sound very similar to me, to the point it might be better to pick just one.

Reply:
I've updated and simplified this section quite a bit.